### PR TITLE
Test if cgroup configuration is propaged to cgroup

### PR DESCRIPTION
**Pull request by kokosing**
Test if cgroup configuration is propaged to cgroup

Setup cluster to use cgroups and test if containers use cgroups
properly.

Reviewers: anu, losipiuk
Task: SWARM-1724

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,7 @@
         <slider.version>0.80.0-incubating</slider.version>
         <junit.version>4.11</junit.version>
         <tempto.version>1.0.37</tempto.version>
+        <hive-jdbc.version>0.13.1-2</hive-jdbc.version>
```
Comment (anusudarsan) [inline pom.xml:67]: thanks :)

File: presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/PrerequisitesClusterFulfiller.groovy [hunk 2]
```diff
@@ -117,7 +118,26 @@ public class PrerequisitesClusterFulfiller
...
   private void restartResourceManager()
   {
-    runOnMaster([RESTART_RM_CMD])
+    runOnMaster(['/etc/init.d/hadoop-yarn-resourcemanager restart'])
+  }
+
+  private void setupCgroup()
+  {
+    runOnAll([
+            'ls /cgroup || yum install -y libcgroup',
```
Comment (anusudarsan) [inline presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/PrerequisitesClusterFulfiller.groovy:127]: these things will need sudo access to the cluster you run tests on, right? Can you update the documentation saying this?

**Comment by anusudarsan** [comment]
lgtm mod comment

**Comment by anusudarsan** [comment]
Also, do you think any of these steps should go to the main README? If so, please add them. Right now we are pointing to the YARN CGroups url for the setup phase. I think that is ok, but if you found anything Slider/Presto specific please add it to the main README.

File: presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/PrerequisitesClusterFulfiller.groovy (hunk 2 above)
Comment (kokosing) [inline presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/PrerequisitesClusterFulfiller.groovy:127]: it is already specified in readme

Comment (kokosing) [inline presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/PrerequisitesClusterFulfiller.groovy:127]: see 

```
set a password for yarn user on master node. In your src/test/resources/tempto-configuration-local.yaml yarn@master password and a private key to connect any cluster node as root user, example settings:
```

**Comment by kokosing** [comment]
It is hard to tell what should be told about cgroups in readme. It is quite complicated and very system and hadoop dependant how it is should be configured. I would leave it as it is.

Also I have a concern if anybody else could run this tests besides us with teradata infrastructure.

**Pull request merged**